### PR TITLE
Review messages v1

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: messages/init/messages.go [hunk 1]
```diff
@@ -3,29 +3,27 @@ package init
...
+	EdgeApplicationsInitUsage = `init [flags]`
 	EdgeApplicationsInitShortDescription  = "Initializes an Edge Application"
 	EdgeApplicationsInitLongDescription   = "Defines primary parameters based on a given name and application type to start an Edge Application on the Azion’s platform"
 	EdgeApplicationsInitRunningCmd        = "Running init step command:\n\n"
 	EdgeApplicationsInitFlagName          = "The Edge application's name"
-	EdgeApplicationsInitFlagTemplate      = "The type of Edge Application"
-	EdgeApplicationsInitFlagMode          = "The mode of Edge Application"
-	EdgeApplicationsInitFlagYes           = "Forces the automatic response 'yes' to all user input"
-	EdgeApplicationsInitFlagNo            = "Forces the automatic response 'no' to all user input"
+	EdgeApplicationsInitFlagTemplate      = "The Edge Application's type. Example: astro"
```
Comment (PatrickMenoti) [inline messages/init/messages.go:11]: Let's change type to preset

File: messages/init/messages.go [hunk 2]
```diff
@@ -3,29 +3,27 @@ package init
...
+	EdgeApplicationsInitNameNotSentSimple = "The project name was not sent through the --name flag; By default, when --name is not given, the working directory is used\n"
+	EdgeApplicationsInitNameNotSentStatic = "The project name was not sent through the --name flag; By default, when --name is not given, the working directory is used\n"
 	EdgeApplicationsUpdateNamePackageJson = "Updating your package.json name field with the value informed through the --name flag\n"
 	EdgeApplicationsInitTypeNotSent       = "The Project Type was not sent through the --template flag; By default when --template is not informed it is auto-detected based on the framework used by the user\n\n"
 	InitDevCommand                        = "If you want to start a local development server later, run 'azion dev'\n"
 	InitDeployCommand                     = "If you want to deploy your project later, run 'azion deploy'\n"
-	InitInstallDeps                       = "Installing projct dependencies"
+	InitInstallDeps                       = "Installing project dependencies"
```
Comment (PatrickMenoti) [inline messages/init/messages.go:28]: 🤡 oops

File: messages/link/errors.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ var (
 	ErrorInstallVulcan                 = errors.New("Failed to install Vulcan: %s")
 	ErrorOpeningConfigFile             = errors.New("Failed to open the config.json file. The file doesn't exist, is corrupted, or has an invalid JSON format. Verify if the file format is JSON or fix its content according to the JSON format specification at https://www.json.org/json-en.html")
 	ErrorUnmarshalConfigFile           = errors.New("Failed to unmarshal the config.json file. Verify if the file format is JSON or fix its content according to the JSON format specification at https://www.json.org/json-en.html")
-	ErrorGetAllTags                    = errors.New("Failed returning all Reference tags in a repository. Verify your repository tags and try again. If the error persists, contact Azion support.")
+	ErrorGetAllTags                    = errors.New("Failed returning all reference tags in a repository. Verify your repository tags and try again. If the error persists, contact Azion support.")
```
Comment (PatrickMenoti) [inline messages/link/errors.go:15]: Isn't "Failed to return" better? Then we follow the same pattern of the other messages

File: messages/link/messages.go [hunk 4]
```diff
@@ -8,16 +8,16 @@ var (
 	EdgeApplicationsLinkLongDescription   = "Defines primary parameters based on a given name and application type to link a Project on the Azion’s platform"
 	EdgeApplicationsLinkRunningCmd        = "Running link step command:\n\n"
 	EdgeApplicationsLinkFlagName          = "The Edge application's name"
-	EdgeApplicationsLinkFlagTemplate      = "The type of Edge Application"
-	EdgeApplicationsLinkFlagMode          = "The mode of Edge Application"
+	EdgeApplicationsLinkFlagTemplate      = "The Edge Application's type"
```
Comment (PatrickMenoti) [inline messages/link/messages.go:11]: type to preset

File: messages/personal-token/messages.go [hunk 5]
```diff
@@ -5,22 +5,22 @@ var (
...
-	CreateLongDescription  = "Creates an personal token"
-	CreateFlagName         = "The flag name"
-	CreateFlagExpiresAt    = "The flag expires at"
-	CreateFlagDescription  = "The flag description"
-	CreateFlagIn           = "Path to a JSON file containing the attributes of the personal token that will be created; you can use - for reading from stdin"
+	CreateLongDescription  = "Creates a personal token to be used for authentication and security"
+	CreateFlagName         = "The personal token's name. It's required if the --in flag is not informed."
+	CreateFlagExpiresAt    = "The personal token's expiration. It's required if the --in flag is not informed."
+	CreateFlagDescription  = "The personal token's description"
+	CreateFlagIn           = "Path to a JSON file containing the attributes of the personal token that being created; you can use - for reading from stdin"
```
Comment (PatrickMenoti) [inline messages/personal-token/messages.go:17]: that are being created

File: messages/link/errors.go (hunk 3 above)
Comment (gabriel-azion) [reply messages/link/errors.go:15]: agreed

File: messages/personal-token/messages.go (hunk 5 above)
Comment (gabriel-azion) [reply messages/personal-token/messages.go:17]: I'll just remove `that`

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**